CMakeLists.txt

```diff
@@ -77,12 +79,15 @@ if(NOT EXISTS "${GLOW_THIRDPARTY_DIR}/onnx")
   message(FATAL_ERROR "No onnx git submodule. Run: git submodule update --init --recursive")
 endif()
 
-add_subdirectory(tests/googletest/)
-
 add_subdirectory(lib)
-add_subdirectory(examples)
 add_subdirectory(tools)
-add_subdirectory(tests)
+if (GLOW_BUILD_EXAMPLES)
```

Comment:
this is also only needed for build examples: find_package(PNG). Would be good just group those things together.

Reply:
I think the PNG package is also needed for the tools (image classifier), so we may need to keep it outside GLOW_BUILD_EXAMPLES

Reply:
Indeed, it's used in image classifier. Disregard my previous comment :)